packages/eas-cli/src/commands/metadata/configure.ts

```diff
@@ -0,0 +1,85 @@
+import { getConfig } from '@expo/config';
+import { Flags } from '@oclif/core';
+
+import EasCommand from '../../commandUtils/EasCommand';
+import { CredentialsContext } from '../../credentials/context';
+import Log, { learnMore } from '../../log';
+import { createMetadataContextAsync } from '../../metadata/context';
+import { downloadMetadataAsync } from '../../metadata/download';
+import { handleMetadataError } from '../../metadata/errors';
+import { RequestedPlatform } from '../../platform';
+import { findProjectRootAsync, getProjectIdAsync } from '../../project/projectUtils';
+import { ensureLoggedInAsync } from '../../user/actions';
+
+type RawCommandFlags = {
+  platform?: string;
+  profile?: string;
+};
+
+type CommandFlags = {
+  requestedPlatforms: RequestedPlatform;
+  profile?: string;
+};
+
+export default class MetadataConfigure extends EasCommand {
+  static hidden = true;
+  static description = 'configure the store configuration file in your project';
+
+  static flags = {
+    platform: Flags.enum({
```

Comment:
We definitely intend to extend this to the play store, but since this only supports ios for now, I think we can leave it out but that might break user's pipelines when we do that.

Reply:
On the other hand, people will complain that they need to pass `-p ios` even though it's always ios

My preference would be to use sth like `metadata:appstore:update` which does not require options or breaking changes if we add new stores, but if you do not want to embed platform in name of the command I think breaking change in the future is better.